pkg/log/log_messages.go

```diff
@@ -80,3 +80,17 @@ const CAKC064 string = "CAKC064 Unable to find Authenticator for this configurat
 const CAKC065 string = "CAKC065 File path is invalid %s"
 const CAKC066 string = "CAKC066 Performing authn-jwt"
 const CAKC067 string = "CAKC067 Failed reading jwt token from %s"
+const CAKC068 string = "CAKC068 Loading config object..."
+const CAKC069 string = "CAKC069 Picking configuration object..."
+const CAKC070 string = "CAKC070 Chosen %q configuration"
+const CAKC071 string = "CAKC071 Gathering input settings for configuration..."
+const CAKC072 string = "CAKC072 Gathered input settings successfully"
```

Comment:
Is this how we said we would do this?
Not "Successfully gathered input settings" ??